source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,51 @@ Sensitive Data Encryption
 Introduction
 ------------
 
-Cras sed rutrum nisi, vitae tristique risus. Aliquam erat volutpat.
-Donec auctor nisl eget ornare accumsan. Pellentesque lectus tortor,
-consectetur molestie pellentesque ac, blandit non libero.
+If you store information in MongoDB such as confidential personal details, payment information, or proprietary data, you are probably interested in protecting your data from unauthorized access. In addition, the data you store may be subject to governance, privacy, and security compliance mandates. 
+
+MongoDB offers several methods that reduce your risk of a data breach resulting from unauthorized access whether accidental or by a malicious act including:
+
+* `Role-based access control <https://docs.mongodb.com/manual/core/authorization/>`_
+* `Monitoring and logging <https://docs.mongodb.com/manual/core/auditing/>`_
+* `TLS/SSL network transport encryption <https://docs.mongodb.com/manual/core/security-transport-encryption/>`_
+* `Data encryption at rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_
+* `Client-side field level encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/>`_
+
+This document guides you through setup and implementation of Client-Side Field Level Encryption (CSFLE), a feature that encrypts document fields specified by a developer, in a example scenario to demonstrate how it can be used to secure your data.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
 
 Problem
 ~~~~~~~
 
-Pellentesque non mi et est congue mattis pretium sit amet mi. Praesent
-nec tortor nec arcu molestie dapibus. Phasellus at nunc ut quam molestie
-finibus. Suspendisse finibus lectus eget nulla eleifend, vel consequat
-leo rhoncus. Phasellus lacinia tortor id aliquet condimentum. Morbi
-sodales est a leo rhoncus, non pretium neque dignissim.
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, appointments, and vital sign measurement reports for a fictitious company, *MedcoMD*. While none of this data should ever be exposed to the public, if a malicious user gained access to data such as the patient's social security number (SSN) or their vital sign measurements, it would be disastrous and damaging to the company.
```

Comment:
**Problem** is the scenario definition, so I don't think we need to lead with "in this scenario". Instead, let's just state the situation:

- The medical company **MedcoMD** stores sensitive patient data such as...
- None of a patient's data should be public, but certain data is particularly private
- It's important for the company _and the patient_ that data is private and secured

Reply:
Adds clarity, thanks.